### PR TITLE
IoT Monitoring tutorial with Prometheus

### PR DESCRIPTION
**Pull request by ptone**

### CODE REVIEW

**Comment by ToddKopriva** [comment]
I'll review this in parallel with the other document that we discussed.

File: tutorials/cloud-iot-prometheus-monitoring/metric-puller/kubernetes/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    metadata:
+      labels:
+        app: metric-puller
+      name: metric-puller
+    spec:
+      containers:
+        - name: metric-puller
+          image: "gcr.io/cloud-solutions-images/metric-puller:0.0.1"
+          imagePullPolicy: Always
+          args: ["--project", "ptone-serverless", "--subscription", "metric-pull"]
```
[the remark this answers is not in the document]
Comment (ptone) [reply tutorials/cloud-iot-prometheus-monitoring/metric-puller/kubernetes/deployment.yaml]: Yeah - a bit of this is semi-orphaned, as the main flow now uses a helm chart which runs the simulator directly.  There isn't any vulnerability exposed in leaving it - maybe a sub readme for that folder? Or just remove it as cruft? (the whole metric-puller dir)

File: tutorials/cloud-iot-prometheus-monitoring/index.md [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+cause higher churn of different series. As a result, Prometheus is now more capable of this kind of approach to device 
+debugging.
+
+## Setting up the workbench
+
+We recommend that you perform these setup steps in the Cloud Shell environment.
+
+### Clone the tutorial repository
+
+```
```
[the remark this answers is not in the document]
Comment (ptone) [reply tutorials/cloud-iot-prometheus-monitoring/index.md]: I'll push a change with the presumed final path in community

File: tutorials/cloud-iot-prometheus-monitoring/metric-puller/kubernetes/deployment.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ptone) [reply tutorials/cloud-iot-prometheus-monitoring/metric-puller/kubernetes/deployment.yaml]: I opted to remove it

File: tutorials/cloud-iot-prometheus-monitoring/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ptone) [reply tutorials/cloud-iot-prometheus-monitoring/index.md]: done

**Pull request merged**